setup.py

```diff
@@ -36,6 +42,7 @@ def run(self):
     name="html2text",
     version=".".join(map(str, __import__('html2text').__version__)),
     description="Turn HTML into equivalent Markdown-structured text.",
+    long_description=read_md('README.md'),
```

Comment:
#116 was an issu on our side and I cannot say that it has been fixed until you push to Pypi again and the description comes out ok. I have closed it but it might reopen on the next version update.

Also I think this should probably not go to the Changelog since it is not something about the software itself.